doc/ws.md

```diff
@@ -0,0 +1,121 @@
+# Websockets interface
+
+## What is this?
+
+This interface is a new way to talk with Orion using websockets, this is point to improve the comunication with a faster and efficient protocol, now we have a persistent connection with a open socket. 
```

Comment:
Appart of efficiency, there is another important aspect to take into account that should be mentionded: Websocket are a way of easing the interaction between Orion and client apllications running in browsers and smartphone applications.